api/src/test/java/com/epam/pipeline/test/creator/user/UserCreatorUtils.java

```diff
@@ -45,6 +69,18 @@ public static PipelineUser getPipelineUser(final String name) {
         return pipelineUser;
     }
 
+    public static PipelineUser getPipelineUser() {
+        PipelineUser pipelineUser = new PipelineUser(TEST_STRING);
```

Comment:
I suppose the overloaded `getPipelineUser` method can be used rather than the constructor call..